test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -813,6 +851,40 @@ UCS_TEST_P(test_uct_cm_sockaddr, conn_to_non_exist_ip)
     }
 }
 
+UCS_TEST_P(test_uct_cm_sockaddr, connect_client_to_server_with_delay)
+{
+    UCS_TEST_MESSAGE << "Testing "     << m_listen_addr
+                     << " Interface: " << GetParam()->dev_name;
+
+    m_delay_conn_reply = true;
+
+    cm_listen_and_connect();
+
+    EXPECT_FALSE(m_cm_state &
+                 (TEST_CM_STATE_SERVER_CONNECTED | TEST_CM_STATE_CLIENT_CONNECTED |
```

Comment:
need to move most of the code in here inside server_handle_delayed_req(), it's common for both tests